tpgtools/main.go

```diff
@@ -207,7 +207,13 @@ func getResources(packagePath string, specs []os.FileInfo) []*Resource {
 			glog.Exit(err)
 		}
 
-		lRaw := schema.Extension["x-dcl-locations"].([]interface{})
+		l := schema.Extension["x-dcl-locations"]
```

Comment:
nit: should this `l` be `lRaw` and the current `lRaw` `locations` or `ls`? `l` is used below to refer to an individual location.

That way we process schema -> array -> location as `lRaw` -> `locations` -> `l` instead of `l` -> `lRaw` -> `l`.